asset_dashboard/static/js/components/map_utils/MapClipper.js

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useState } from 'react'
+import { FeatureGroup } from 'react-leaflet'
+import clip from 'turf-clip'
+import EditControl from './EditControl'
+
+export default function MapClipper({ geoJson, onClipped }) {
+  const [drawnGeometries, setDrawnGeometries] = useState(null)
```

Comment:
ideas for a better variable name? kinda awkward

Reply:
i had `selectedGeometries` but that was confusing since "selected geometries" could be interpreted as the geometries that were selected from the underlying geojson, and not the geometries created by the draw. even typing that out is confusing.